src/main/tests/models/PCAPTest.java

```diff
@@ -0,0 +1,5 @@
+package models;
+
+public class PCAPTest {
```

Comment:
what is this class for btw?

Reply:
Scaffolding for Melba/Erik's stuff I think.